src/util/animate.js

```diff
@@ -52,14 +115,13 @@
           return;
         }
         if (abort(current, valuePerc, timePerc)) {
-          // remove this in 4.0
-          // does to even make sense to abort and run onComplete?
-          onComplete(endValue, 1, 1);
```

Comment:
my comment still stands

Reply:
since we did not remove this in 4.0 we can't remove it now. now it has to wait for 5.0

Reply:
af7cdc3f9cff33cd74e83b77ac479273269ad59d